miso-web/src/main/webapp/scripts/list_sequencer.js

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2012. The Genome Analysis Centre, Norwich, UK
+ * MISO project contacts: Robert Davey @ TGAC
+ * *********************************************************************
+ *
+ * This file is part of MISO.
+ *
+ * MISO is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MISO is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MISO.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * *********************************************************************
+ */
+
+ListTarget.sequencer = {
+  name : "Sequencer",
+  createUrl : function(config, projectId) {
+    return "/miso/rest/sequencer/dt";
+  },
+  createBulkActions : function(config, projectId) {
+    return [];
+  },
+  createStaticActions : function(config, projectId) {
+    if (config.isAdmin) {
+      return [ {
+        "name" : "Add",
+        "handler" : function() {
+          
+          Utils.showDialog("Add Sequencer", "Add", [ {
+            property : "name",
+            type : "text",
+            label : "Name"
+          }, {
+            property : "platform",
+            type : "select",
+            values : Constants.platforms,
+            getLabel : function(platform) {
+              return platform.platformType + " - " + platform.instrumentModel;
+            },
+            label : "Platform"
+          }, {
+            property : "serialNumber",
+            type : "text",
+            label : "Serial Number"
+          }, {
+            property : "ip",
+            type : "text",
+            label : "Hostname/IP Address"
+          }, {
+            property : "dateCommissioned",
```

Comment:
A date picker attached to a text field like we use on most forms would be clearer. This is an optional field and while no date is selected initially, it looks like the current date is. There doesn't seem to be a way to deselect a date after selecting one by accident either